src/raw/cgroup_detect.go

```diff
@@ -1,17 +1,119 @@
 package raw
 
 import (
+	"bufio"
+	"fmt"
 	"io"
 	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/newrelic/infra-integrations-sdk/log"
 )
 
 const (
-	mountsFilePath    = "/proc/mounts"
-	cgroupFilePathTpl = "/proc/%d/cgroup"
+	mountsFilePath           = "/proc/mounts"
+	cgroupFilePathTpl        = "/proc/%d/cgroup"
+	cgroup1MountName         = "cgroup"
+	cgroup2MountName         = "cgroup2"
```

Comment:
Nit: for consistency with the rest of the codebase I will rename them to something like `cgroupv1.*` and `cgroupv2.*`

Reply:
Done